crates/common_enums/src/enums.rs

```diff
@@ -2141,6 +2142,35 @@ pub enum PayoutType {
     Wallet,
 }
 
+#[derive(
+    Clone,
+    Copy,
+    Debug,
+    Default,
+    Eq,
+    Hash,
+    PartialEq,
+    serde::Deserialize,
+    serde::Serialize,
+    strum::Display,
+    strum::VariantNames,
+    strum::EnumIter,
+    strum::EnumString,
+    ToSchema,
+)]
+#[router_derive::diesel_enum(storage_type = "db_enum")]
+#[serde(rename_all = "snake_case")]
+#[strum(serialize_all = "snake_case")]
+pub enum PayoutMethodType {
```

Comment:
Is it possible to avoid adding this list? And instead derive the payout type from `PayoutMethodData`? We can make use of `PaymentMethodType` if needed, and use TryFrom to convert to and from `PayoutMethodData`